### PR TITLE
Clean Up Art/UI/Interactions+Containers

### PR DESCRIPTION
**Pull request by cosmiccoincidence**
## Summary

Cleans up the Interactions and Containers folders under /Art/UI/.

- Renames several files using PascalCase.
- Re organizes both folders above.
- Also renames and moves some of the folder from the Misc folder.

### CODE REVIEW

**Review by stilnat — CHANGES_REQUESTED** [review]
wait for #995 before merge.

**Comment by cosmiccoincidence** [comment]
Wait for #1021 to merge first!

**Comment by cosmiccoincidence** [comment]
A lot of what I did was done by John in recent Asset Pr. I will close this one.

**Pull request closed without merging**
